src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -50,18 +69,23 @@ const tableFields = {
   lastModified: 'lastModified'
 }
 
-const KEY_ANALYSES_SORT_ORDER = 'AnalysesSortOrder'
+export const KEY_ANALYSES_SORT_ORDER: string = 'AnalysesSortOrder'
 
-const noWrite = 'You do not have access to modify this workspace.'
+const noWrite: string = 'You do not have access to modify this workspace.'
 
-const sortTokens = {
+interface Sort {
```

Comment:
Similarly, it doesn't look there's a need to declare this interface separately. I think the necessary types would be provided by:
```ts
const sortTokens = {
  name: (notebook: AnalysisFile) => notebook.name.toLowerCase()
}
```